packages/commonwealth/client/scripts/App.tsx

```diff
@@ -20,7 +22,18 @@ const Splash = () => {
   );
 };
 
-OpenFeature.setProvider(new InMemoryProvider(featureFlags));
+const unleashConfig = {
+  url: process.env.UNLEASH_FRONTEND_SERVER_URL,
+  clientKey: process.env.UNLEASH_FRONTEND_API_TOKEN,
+  refreshInterval: 15,
+  appName: 'commonwealth-web',
+};
+
+OpenFeature.setProvider(
+  process.env.UNLEASH_FRONTEND_API_TOKEN
```

Comment:
If FRONTEND_API_TOKEN is present in the .env, we will use the unleash server for feature flagging. Otherwise it will default to our regular .env feature flags

Reply:
please move this out from App.tsx file to some config file

Reply:
This set provider will need to go here though, as it will need to be set at the beggining of the entrypoint of the app. I have moved the config to a separate file though.